nebullvm/operations/base.py

```diff
@@ -3,17 +3,17 @@
 
 from loguru import logger
 
-from nebullvm.tools.base import Device
+from nebullvm.tools.base import Device, DeviceType
 from nebullvm.tools.feedback_collector import FeedbackCollector
 from nebullvm.tools.utils import gpu_is_available
 
 
 def _check_device(device: Optional[str]) -> Device:
```

Comment:
what if the input string is "gpu:1"? Is this case supported?

Reply:
I forgot to change this function, it can be deleted because it can be imported from `nebullvm.tools.utils import check_device`